Description: Add test script
Add a test script, testing.sh to echo message. And the script would be
executed right after this package is installed. 
.
 mmc-utils (0+git20191004.73d6c59-2) unstable; urgency=medium
 .
   * d/control
     - Update upstream URL. (Closes: #997966)
       Thanks to Andy Isaacson <adi@hexapodia.org>.
   * d/copyright
     - Update to 2022.
Author: Nobuhiro Iwamatsu <iwamatsu@debian.org>
Bug-Debian: https://bugs.debian.org/997966

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2022-11-24

--- /dev/null
+++ mmc-utils-0+git20191004.73d6c59/testing.sh
@@ -0,0 +1,2 @@
+#!/bin/bash
+echo “this is a test from Yen-Po Wang ”
